docs/index.rst

```diff
@@ -57,6 +57,12 @@ The vision is that it grows over time as the teams and governance structure evol
    policies/request_for_proposals_(RFP)_process
    policies/recognizing_contributors_sponsors_and_partners
 
+.. toctree::
+   :maxdepth: 2
+   :caption: On-boarding
+   :hidden:
+
```

Comment:
```suggestion

   onboarding/onboarding
```

You need to add the link here to the file, making the files is throwing an error.

Reply:
Yeah, I missed that, thank you